src/scripts/eastern_kingdoms/tirisfal_glades/scarlet_monastery/instance_scarlet_monastery.cpp

```diff
@@ -341,24 +348,31 @@ struct instance_scarlet_monastery : ScriptedInstance
                         break;
                     case EVENT_TALK1:
                         if (m_uiAshbringerWielderGUID)
-                            if (Player* pPlayer = GetMap()->GetPlayer(m_uiAshbringerWielderGUID))
+                            if (Player* pPlayer = instance->GetPlayer(m_uiAshbringerWielderGUID))
                                 DoScriptText(SAY_COMMANDER1, pMograine, pPlayer);
-                        m_events.ScheduleEvent(EVENT_SUMMON, Seconds(10));
                         break;
                     case EVENT_SUMMON:
-                        if (Creature* pC = pMograine->SummonCreature(NPC_HIGHLORD_MOGRAINE, 1065.130737f, 1399.350586f, 30.763723f, 6.282961f, TEMPSUMMON_TIMED_DESPAWN, 400000))
+                        if (Creature* pC = pMograine->SummonCreature(NPC_HIGHLORD_MOGRAINE, 1034.9252f, 1399.0653f, 27.393204f, 6.257956981658935546f, TEMPSUMMON_TIMED_DESPAWN, 400000))
                         {
                             pC->SetFactionTemporary(35, TEMPFACTION_RESTORE_RESPAWN);
-                            pC->SetObjectScale(2.0f);
-                            pC->CastSpell(pC, SPELL_MOGRAINE_COMETH, true);
+                            pC->SetDisplayId(16180);
+                            pC->SetVirtualItem(BASE_ATTACK, EQUIP_UNEQUIP);
+                            pC->SetObjectScale(2.0);
+                            pC->CastSpell(pC, SPELL_MOGRAINE_COMETH, false);
                             // TODO: The coordinates below appear to be guessed, set the correct ones once we have them.
```

Comment:
is this comment still relevant?

Reply:
When the Highlord Mograine is first generated, the model is displayed as 16180. There are no weapons in the hands. In the same timestamp, he used spell 28688 and there is no difference between spell 28688 true and false Is there anything wrong with that？